app/README.md

```diff
@@ -1,16 +1,186 @@
 
-To build a redistributable, production mode package, use `wails build`.
+[![Go](https://img.shields.io/badge/Go-00ADD8?style=for-the-badge&logo=go&logoColor=white)](https://go.dev/)
+[![Svelte](https://img.shields.io/badge/Svelte-FF3E00?style=for-the-badge&logo=svelte&logoColor=white)](https://svelte.dev/)
+[![TypeScript](https://img.shields.io/badge/TypeScript-3178C6?style=for-the-badge&logo=typescript&logoColor=white)](https://www.typescriptlang.org/)
+[![Wails](https://img.shields.io/badge/Wails-FF6C37?style=for-the-badge&logo=wails&logoColor=white)](https://wails.io/)
+[![Bun](https://img.shields.io/badge/Bun-000000?style=for-the-badge&logo=bun&logoColor=white)](https://bun.sh/)
+[![Vite](https://img.shields.io/badge/Vite-646CFF?style=for-the-badge&logo=vite&logoColor=white)](https://vitejs.dev/)
+[![SCSS](https://img.shields.io/badge/SCSS-CC6699?style=for-the-badge&logo=sass&logoColor=white)](https://sass-lang.com/)
+[![Linux](https://img.shields.io/badge/Linux-FCC624?style=for-the-badge&logo=linux&logoColor=black)](https://kernel.org/)
+[![Windows](https://img.shields.io/badge/Windows-0078D6?style=for-the-badge&logo=windows&logoColor=white)](https://microsoft.com/windows)
+
+---
+
+## Table of Contents
+
+- [Features](#features)
+- [Demo](#demo)
+- [Installation](#installation)
+  - [Linux](#linux)
+  - [Windows](#windows)
+  - [Building from Source](#building-from-source)
+  - [Build Steps](#build-steps)
+- [License](#license)
+- [Author](#author)
+
+## Features
+
+- **Real-Time USB Device Tree**: Visualize connected USB devices in a hierarchical tree structure
+  - Tree shows device manufacturer and product names
+  - Devices negotiated speed are displayed to the right of device names
+- **Hot-Plug Detection**: Automatically detect when devices are connected or disconnected
+  - changes are reflected in the tree structure with color coding and icons
+  - all changes are logged to a log section at the bottom of the app
+  - Device tree can be reset to the current state by clicking the "Refresh" button
+- **Device Details**: device tooltip shows comprehensive information including:
+  - Vendor ID and Product ID
+  - Device bus information
+  - Clicking a device in the tree will open a database search for more information.
+- **Cross-Platform**: Native support for Linux and Windows
+  - Linux versions available for x86_64 and ARM64
+  - Windows version available for x86_64
+- **Modern UI**: Clean, responsive interface built with Svelte and Carbon Design System
+  - Dark mode supported
+
+## Demo:
+
+<img src="/images/output.gif" alt="Demo" width="500" >
+
+## Installation
+
+### Linux
+
+#### From Release
+
+1. Download the latest `usb-tree-linux-amd64.tar.gz` or `usb-tree-linux-arm64.tar.gz` from the
+   [releases page](https://github.com/AOzmond/usb-tree/releases)
+2. Extract the archive to your desired location:
+   ```bash
+   tar -xzf usb-tree-linux-amd64.tar.gz
+   ```
+3. Run binary:
+   ```bash
+   ./usb-tree
+   ```
+
```

Comment:
```suggestion

1. Download the latest `usb-tree-linux-amd64.tar.gz` or `usb-tree-linux-arm64.tar.gz` from the
   [releases page](https://github.com/AOzmond/usb-tree/releases)
   
2. Extract the archive to your desired location:
   ```bash
   tar -xzf usb-tree-linux-amd64.tar.gz
   ```

3. Run binary:
   ```bash
   ./usb-tree
   ```

```
GitHub's "suggestion" formatting is super broken if there's code blocks in them. You have empty lines between bullets in Build Steps